_local_envs_for_test.sh

```diff
@@ -4,14 +4,11 @@
 
 # Need to be set for tests to run
 export MASTER_ADDR=localhost
-export MASTER_PORT=29500
```

Comment:
12345 seems like a more reasonable default

Reply:
ah but right now only `29500` works for me 😭

Reply:
Why do we even need to set these? Can't we just run with defaults?

Reply:
Yeah these are still required, but we could just set them in conftest.py